python/brayns/instance/jsonrpc/json_rpc_client.py

```diff
@@ -37,6 +38,9 @@ def __init__(self, websocket: WebSocket, logger: logging.Logger) -> None:
         self._handler = JsonRpcHandler(self._manager, logger)
         self._dispatcher = JsonRpcDispatcher(self._handler)
 
+    def __contains__(self, id: Union[int, str]) -> bool:
```

Comment:
Just a personal preference, using the __contains__ to have the sintactic sugar of id in _client makes the client object to look like an id container, and does not make it obvious that you are checking for used message ids. I'd use a specific method that tell right away what is being check for.

However, is not an issue so you can merge.

Reply:
Yeah you are right I will do it in another PR.